esp_server/PDSClient/ConnectionManager/DBConnect.cs

```diff
@@ -556,7 +562,7 @@ public List<PhoneInfo> GetLastMinuteData(int nBoards, int threshold = 0)
                             }
                         }
                     }
-                    list = ReduceList(list, ReduceType.Mean);
+                    //list = ReduceList(list, ReduceType.Mean);

```

Comment:
non commentare il vecchio codice, cancellalo direttamente. se ci servira' di nuovo, possiamo sempre recuperarlo da git

Reply:
quando lo commento è perchè non mi serve al momento, e fino a quando non mi funziona la mia roba non lo tolgo, mi serve capire come funziona bene tutto, comunque questo è insieme alla query, non le avevo ricontrollate e non era in programma di metterle insieme a tutto, quindi piccola dimenticanza